test/util/validationUtils.test.js

```diff
@@ -279,6 +302,35 @@ describe('validateCareGapsParams', () => {
     }
   });
 
+  test('validateCareGapsParams does not throw error with practitioner instead of subject', async () => {
+    const VALID_REQ = {
+      query: VALID_PRACTITIONER_QUERY
+    };
+    expect(validateCareGapsParams(VALID_REQ.query)).toBeUndefined();
+  });
+  test('validateCareGapsParams throws error with both practitioner and subject', async () => {
+    try {
+      validateCareGapsParams(SUBJECT_AND_PRACTITIONER_QUERY);
+      throw new Error('validateCareGapsParams failed to throw an error when provided both subject and practitioner');
```

Comment:
Also my bad, but instead of throwing an error here, use expect.fail() with the same message. Will make debugging easier